contracts/number/Math.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Let's name this `NumberUtils.sol` we do have some precedent with that scheme elsewhere in the `root` codebase - and then we don't have to overload the naming when we import.